engines/mohawk/configure.engine

```diff
@@ -2,5 +2,5 @@
 # add_engine [name] [desc] [build-by-default] [subengines] [base games] [deps]
 add_engine mohawk "Mohawk" yes "cstime myst riven" "Living Books" "highres"
 add_engine cstime "Where in Time is Carmen Sandiego?" no
-add_engine riven "Riven: The Sequel to Myst" no "" "" "16bit"
+add_engine riven "Riven: The Sequel to Myst" yes "" "" "16bit"
```

Comment:
This is on purpose. I'd like Riven to be part of the 1.10 release, possibly publishing the testing announcement a few weeks after Starship Titanic.